### ISSUE TITLE
build fails because of missing natend repo

### ISSUE THREAD

**Issue by emilio** [mozilla-mobile/mozilla-vpn-client#3741]
```
[ 40%] Generating netfilter.a, netfilter.h
cd /home/emilio/src/moz/mozilla-vpn-client/linux/netfilter && /usr/bin/cmake -E env GOCACHE=/home/emilio/.cache/go-build go build -buildmode=c-archive -v -o /home/emilio/src/moz/mozilla-vpn-client/build/src/netfilter.a netfilter.go
go: github.com/google/nftables@v0.0.0-20210514154851-a285acebcad3 requires
	github.com/koneu/natend@v0.0.0-20150829182554-ec0926ea948d: invalid version: git ls-remote -q origin in /home/emilio/go/pkg/mod/cache/vcs/1a9251955b480a9a40583c1db9751cccef97993062f8285e6f0904b90d80152a: exit status 128:
	ERROR: Repository not found.
	fatal: Could not read from remote repository.
	
	Please make sure you have the correct access rights
	and the repository exists.
go: github.com/google/nftables@v0.0.0-20210514154851-a285acebcad3 requires
	github.com/koneu/natend@v0.0.0-20150829182554-ec0926ea948d: invalid version: git ls-remote -q origin in /home/emilio/go/pkg/mod/cache/vcs/1a9251955b480a9a40583c1db9751cccef97993062f8285e6f0904b90d80152a: exit status 128:
	ERROR: Repository not found.
	fatal: Could not read from remote repository.
	
	Please make sure you have the correct access rights
	and the repository exists.
```

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/VPN-2400)

**Comment by Oscar-21** [comment]
I am having this exact same issue building on Fedora 36.

## First Build Attempt
```bash
   Compiling mozillavpnnp v0.1.0 (/home/user/Documents/mozilla-vpn-client/extension/bridge)
go: github.com/google/nftables@v0.0.0-20210514154851-a285acebcad3 requires                                                                  
	github.com/koneu/natend@v0.0.0-20150829182554-ec0926ea948d: invalid version: git ls-remote -q origin in /home/user/go/pkg/mod/cache/vcs/1a9251955b480a9a40583c1db9751cccef97993062f8285e6f0904b90d80152a: exit status 128:
	fatal: could not read Username for 'https://github.com': terminal prompts disabled
Confirm the import path was entered correctly.
If this is a private repository, see https://golang.org/doc/faq#git_https for additional information.
go: downloading github.com/google/nftables v0.0.0-20210514154851-a285acebcad3
go: downloading golang.org/x/sys v0.0.0-20210611083646-a4fc73990273
    Finished release [optimized] target(s) in 1m 57s
[ 40%] Built target cargo_mozillavpnnp
go: github.com/google/nftables@v0.0.0-20210514154851-a285acebcad3 requires
	github.com/koneu/natend@v0.0.0-20150829182554-ec0926ea948d: invalid version: git ls-remote -q origin in /home/user/go/pkg/mod/cache/vcs/1a9251955b480a9a40583c1db9751cccef97993062f8285e6f0904b90d80152a: exit status 128:
	fatal: could not read Username for 'https://github.com': terminal prompts disabled
Confirm the import path was entered correctly.
If this is a private repository, see https://golang.org/doc/faq#git_https for additional information.
gmake[2]: *** [src/CMakeFiles/mozillavpn_autogen.dir/build.make:77: src/netfilter.a] Error 1
gmake[1]: *** [CMakeFiles/Makefile2:385: src/CMakeFiles/mozillavpn_autogen.dir/all] Error 2
```
## Things I tried
#### 1. Set up ssh with my github account
set up ssh keys with my local machine and github account.

#### 2. Allow go to use terminal prompt
`export GIT_TERMINAL_PROMPT=1`
After this I no longer saw `terminal prompts disabled` in the build errors and was asked for a password from git,

#### 3.  Use ssh as default connection method to git
`git config --global url.ssh://git@github.com/.insteadOf https://github.com/`

#### Final unsolved build error
```bash
go: github.com/google/nftables@v0.0.0-20210514154851-a285acebcad3 requires
	github.com/koneu/natend@v0.0.0-20150829182554-ec0926ea948d: invalid version: git ls-remote -q origin in /home/user/go/pkg/mod/cache/vcs/1a9251955b480a9a40583c1db9751cccef97993062f8285e6f0904b90d80152a: exit status 128:
	ERROR: Repository not found.
	fatal: Could not read from remote repository.
	
	Please make sure you have the correct access rights
	and the repository exists.
go: github.com/google/nftables@v0.0.0-20210514154851-a285acebcad3 requires
	github.com/koneu/natend@v0.0.0-20150829182554-ec0926ea948d: invalid version: git ls-remote -q origin in /home/user/go/pkg/mod/cache/vcs/1a9251955b480a9a40583c1db9751cccef97993062f8285e6f0904b90d80152a: exit status 128:
	ERROR: Repository not found.
	fatal: Could not read from remote repository.
	
	Please make sure you have the correct access rights
	and the repository exists.
gmake[2]: *** [src/CMakeFiles/mozillavpn_autogen.dir/build.make:77: src/netfilter.a] Error 1
gmake[1]: *** [CMakeFiles/Makefile2:385: src/CMakeFiles/mozillavpn_autogen.dir/all] Error 2
gmake: *** [Makefile:136: all] Error 2
```

**Comment by kkaess** [comment]
I have this problem building on Fedora 36 Workstation for aarch64, but not on x86_64, which does not attempt to generate netfilter.a.

**Comment by mdillavou** [comment]
Same problem on Fedora 36 aarch64. Here is what I did:

```
cd linux/netfiler
edit `go.mod` and delete line: `github.com/google/nftables v0.0.0-20210514154851-a285acebcad3`
Run: `go get github.com/google/nftables`
```

**Comment by oskirby** [comment]
This should be fixed by #3898
